ember-autofocus-modifier/src/modifiers/autofocus.ts

```diff
@@ -0,0 +1,82 @@
+import { modifier } from "ember-modifier";
+import { next } from "@ember/runloop";
+
+const focusableElements = [
+  "BUTTON",
+  "SUMMARY",
+  "IFRAME",
+  "INPUT",
+  "SELECT",
+  "TEXTAREA",
+] as const;
+
+const DEFAULT_SELECTOR =
+  "input:not([disabled]):not([readonly]),textarea:not([disabled]):not([readonly])";
+
+interface ModifierArgs {
+  Element: HTMLElement;
+  Args: {
+    Positional: [string?] | undefined; // Optional selector
+    Named: {
+      disabled?: boolean; // Optional 'disabled' parameter
+    };
+  };
+}
+
+export default modifier<ModifierArgs>(function autofocus(
+  element: HTMLElement,
+  [selector] = [DEFAULT_SELECTOR],
+  { disabled } = { disabled: false },
+) {
+  if (disabled) {
+    return;
+  }
+
+  // Instead of selecting a child element by default, should this (in a major),
+  // be a behavior that is changed via a passed flag?
```

Comment:
Maybe we can discuss this improvement outside this PR (and so remove the comment)? We can create an issue to keep track of it

Reply:
This was in the original `.js` file so I decided to keep it 👀 
To me makes sense to create an issue and remove the comment, will do

Reply:
Without any additional argument, it should select the first child focusable (and not disabled) element if the element holding the modifier is not focusable.
I don't think updating this default behavior is a good thing. On the other hand, improving the API to give more flexibility is probably a good idea.
I'd say let's create the issue, and if we feel like there is room for discussion we will use it to discuss. How does it sound?

Reply:
Sounds good, here I opened the issue and tried to explain the problem: https://github.com/qonto/ember-autofocus-modifier/issues/457

What do you think?

Reply:
It's clear to me! Thanks. I'll let you remove the comment in the code and resolve this thread 🙏